README.md

```diff
@@ -0,0 +1,36 @@
+# Spike Detector Fuse for NEST
+This module monitors the spiking activity of the network, and throws a python exception if the spiking activity goes
+beyond the given threshold for the given length of time.
+
+It is a module for the [NEST simulator](http://www.nest-simulator.org).
+
```

Comment:
It would be cool if you could point them to the comment in 'spike_detector_fuse.h' for dicumentation regarding the fusing mechanics.